docs/exploring-es-data.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[exploring-es-data]]
+= Exploring data in Elasticsearch
+
+[partintro]
+--
+By default Elastic APM data are stored in different index types. 
+
+Indices storing monitored data are built in the format of `apm-[version]-[type]-[date]` by default. 
```

Comment:
monitored data sounds odd but I'm not sure what else to call the onboarding docs

Reply:
Not sure you need to say "monitored data" here. Maybe just say something like:
```
The indices are named `apm-[version]-[type]-[date]` by default.
```

Then give an example. The way it's worded right now, sounds like the indices themselves use some kind of special format, but really it's jut the index name that uses the format.